### PR TITLE
Created update profile component

### PR DESCRIPTION
**Pull request by frankmarra**
This PR creates a component UpdateProfile that allows a user to update their avatar, location, and public page password protection.

This solves the issue that we were having with uploading album artwork in the last PR.

### CODE REVIEW

File: components/Account/Account.jsx [hunk 1]
```diff
@@ -4,121 +4,107 @@ import styles from "./Account.module.css"
...
 
 export default function Account({ session }) {
   const supabase = useSupabaseClient()
   const user = useUser()
   const [loading, setLoading] = useState(true)
-  const [username, setUsername] = useState(null)
-  const [avatarUrl, setAvatarUrl] = useState(null)
+  // const [username, setUsername] = useState(null)
+  // const [avatarUrl, setAvatarUrl] = useState(null)
+  const [updateView, setUpdateView] = useState(false)
```
Comment (hexagoncircle) [inline components/Account/Account.jsx]: Naming: `showUpdateView, setShowUpdateView`—should reflect the action

File: components/Avatar/Avatar.jsx [hunk 2]
```diff
@@ -3,95 +3,81 @@ import { useSupabaseClient } from "@supabase/auth-helpers-react"
...
+import AddImage from "../AddImage/AddImage"
 
-export default function Avatar({ uid, url, size, onUpload }) {
-  const supabase = useSupabaseClient()
-  const [avatarUrl, setAvatarUrl] = useState(null)
-  const [uploading, setUploading] = useState(false)
+export default function Avatar({ url, size }) {
+  // const supabase = useSupabaseClient()
+  // const [imageUrl, setImageUrl] = useState(null)
+  // const [uploading, setUploading] = useState(false)
```
Comment (hexagoncircle) [inline components/Avatar/Avatar.jsx]: Do we need to keep things that are commented out? You can always find this code in a previous commit if you need it.

File: components/Releases/CreateRelease.jsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import slugify from "slugify"
 import { useState } from "react"
 import { useUser, useSupabaseClient } from "@supabase/auth-helpers-react"
+import AddImage from "../AddImage/AddImage"
```
Comment (hexagoncircle) [inline components/Releases/CreateRelease.jsx]: I wish the auto-import would use the aliases. `@/components/AddImage/AddImage`

**Review by hexagoncircle — APPROVED** [review]
👍

**Comment by frankmarra** [comment]
merged

**Pull request merged**